app/views/categories/sessions_controller0.rb

```diff
@@ -1,27 +1,23 @@
+# frozen_string_literal: true
 
 class SessionsController < ApplicationController
```

Comment:
Why do we have a duplicate controller file in views? `app/views/categories/sessions_controller0.rb`?